yolox/exp/yolox_base.py

```diff
@@ -43,7 +43,7 @@ def __init__(self):
         # name of annotation file for evaluation
         self.val_ann = "instances_val2017.json"
         # name of annotation file for testing
-        self.test_ann = "instances_test2017.json"
+        self.test_ann = "image_info_test-dev2017.json"
```

Comment:
`instances_test2017.json` is used for coco-test. Why modifiying it here?  You could use cli to modify it for your need.

Reply:
Sure, I updated the patch and pls check it again